### ISSUE TITLE
Dropbox with multiple machines

### ISSUE THREAD

**Issue by erik-wgp** [ttscoff/nv#547]
I run nvAlt with plain text storage on Dropbox, no simplenote syncing.  Is it a problem to run it like this on multiple Macs concurrently?  There would not really be concurrent updates to individual notes.

If I understand it right, there is a database used for indexing (even in plain text storage mode), and so it would be a common case for a running nvAlt to find the DB updated by the other copy, and I guess a very rare but non-zero chance that two copies of nvAlt would update the DB at the same time.

I understand with any syncing wacky things can happen, but want to avoid anything known to be risky or problematic.

#321 touches on this but it is old and not entirely clear what it means for my use case.

Thanks

**Comment by whatsnewsisyphus** [comment]
Yes there are still problems, I used it like that for a long time but then had a couple notes disappear or get mangled in the process where one machine unsynced it out of existence. I'm guessing what happened was that I had both computers open, as the laptop I often just put to sleep, added something important in the desktop, and laptop synced the previous state of the database up when I closed the app.

A suggestion would be to check if the file is the same before writing and ask whether you want to load the new version.

I'm currently using a different app because of this but would really prefer to come back to nvalt.
